src/topology/algebra/field.lean

```diff
@@ -105,7 +105,7 @@ lemma units_top_group : topological_group Kˣ :=
 local attribute [instance] units_top_group
 
 lemma continuous_units_inv : continuous (λ x : Kˣ, (↑(x⁻¹) : K)) :=
-(topological_ring.induced_units.continuous_coe K).comp topological_group.continuous_inv
+(topological_ring.induced_units.continuous_coe K).comp continuous_id.inv
```

Comment:
```suggestion
(topological_ring.induced_units.continuous_coe K).comp continuous_inv
```
should work here too, right?

Reply:
Putting `@[protect_proj]` on `topological_division_ring` might be enough to resolve the name conflict until you make that `extends` fix.

Reply:
done